.github/workflows/cloud-deployment-test.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
 
-  could-deployment-test:
+  could_deployment_test:
```

Comment:
could you please change to - cloud (since you already touched this code)

Reply:
Done. Thank you.